controllers/provider-gcp/charts/provider-gcp/values.yaml

```diff
@@ -19,9 +19,15 @@ disableControllers:
 disableWebhooks:
 - controlplane
 - controlplaneexposure
+- controlplanebackup
 
 config:
   machineImages:
   - name: coreos
     version: 2023.5.0
     image: projects/coreos-cloud/global/images/coreos-stable-2023-5-0-v20190312
+  etcd:
+    storageClass: gardener.cloud-fast
```

Comment:
To avoid confusion I would also change the values API to match the new ConfigMap API.

Reply:
Done.